scripts/requirements-test.yml

```diff
@@ -18,8 +18,8 @@ channels:
   - defaults
 dependencies:
   # base (vis. recipe/conda_build_config.yaml)
-  - python=3.10
-  - numpy=1.24
+  - python >=3.10
+  - numpy >=1.23
```

Comment:
or potentially

```suggestion
  - python >=3.10,<3.13
  - numpy >=1.23,<1.27
```

Reply:
It's not every combination within there though hopefully conda can figure that out in practise. Can a user specify the python version on the command line or would that need a `python = x.x`?